src/js/common/browserInfoHelper.js

```diff
@@ -106,3 +107,7 @@ export const getBrowserDoNotTrackStatus = () => {
       windowVar.external.msTrackingProtectionEnabled());
   return isBrowserDoNotTrack ? "true" : "false";
 };
+
+export const getDeviceId = () => {
+  return uuid();
```

Comment:
I don't think this is the right location. UUID generation is nothing to do with getting data out of the browser API's. Suggest its own helper file or no util file at all.

Reply:
resolved